src/Components/Presentational/Chrome/Navigation.js

```diff
@@ -98,7 +100,10 @@ const Navigation = () => {
         <AvatarListItem />
       </Drawer>
       <main>
-        <PageWrapper />
+        <Switch>
+          <Route exact path="/" component={PageWrapper} />
+          <Route path="/RequestAsset" component={RequestAssetsContainer} />
```

Comment:
The Information Architecture diagram on Figma shows slightly different routes.
`/search/{query?=params}` and `/search/access/{hash}`
ref:  https://www.figma.com/file/JRTYL35e6a27C3PaP3h4xp/Kirby-UI-v2.0?node-id=182%3A2233

Reply:
Cuz our first iteration of the design was only `/RequestAssets`. 
This one recently get changed.
`Hashing` we need to create a ticket for that. Need more though on this. 
We have to create a ticket to add `queryparam` and `hash`

Reply:
I've documented a story with Ryan.  Since we have documented the tech debt here, I'll resolve this for this PR.